src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp

```diff
@@ -190,98 +194,111 @@ void medAbstractDatabaseImporter::importFile ( void )
     bool atLeastOneImportSucceeded = false;
     bool atLeastOneImportError = false;
 
+    //TODO: reading and filtering represents 50% of the importing process?
```

Comment:
i think you can remove the todo comment. Did you feel, testing the import, that this percentage is too high or slow?

Reply:
If you have a few but big files, it's too high.
If you have a lot of small files, it's too low. 
I don't really have an answer.